server/api/db/sqldb/db.py

```diff
@@ -1398,6 +1402,8 @@ def _find_artifacts(
         attach_tags: bool = False,
         limit: int = None,
         with_entities: list[Any] = None,
+        page: typing.Optional[int] = None,
+        page_size: typing.Optional[int] = None,
```

Comment:
I think that we need to block requests requesting both limit and page size.
the limit is mainly used by ui to reduce the amount of artifacts they requested. so now that the endpoint is paginated, they could use that instead.
what I suggest is that
if limit is given - use as usual
if limit is given with page size, raise conflict and message should say that page_size and limit are conflicting and only one can be given